CHANGELOG.md

```diff
@@ -53,6 +53,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * `jj git fetch` now automatically prints new remote branches and tags by default.
 
+* `--verbose(-v)` is now `--debug` (no short option since it's not intended to be used often)
```

Comment:
nit: we normally write the long and short options separated by a slash (`--verbose/-v`)

btw, further down in this file, we have this old entry:

> The new global flag `-v/--verbose` will turn on debug logging

So we described it as turning on debug logging from the beginning :)